tests/network-tests/src/flows/content/updateApp.ts

```diff
@@ -0,0 +1,113 @@
+      assert.equal(appsByName?.[0]?.oneLiner, newAppMetadataOwnedByMember.oneLiner)
+      assert.equal(appsByName?.[0]?.description, newAppMetadataOwnedByMember.description)
+      assert.deepEqual(appsByName?.[0]?.platforms, newAppMetadataOwnedByMember.platforms)
+    }
+  )
+
+  const newAppCreatedByMemberId = appsCreatedByMember?.[0]?.id
+  if (newAppCreatedByMemberId) {
+    await api.updateApp(member.memberId, newAppCreatedByMemberId, updatedMetadata)
+  }
+
+  await query.tryQueryWithTimeout(
+    () => query.getAppsByName(newAppNameOwnedByMember),
+    (appsByName) => {
+      assert.equal(appsByName?.[0]?.id, newAppCreatedByMemberId)
+      assert.equal(appsByName?.[0]?.name, newAppNameOwnedByMember)
+      assert.equal(appsByName?.[0]?.useUri, newAppMetadataOwnedByMember.useUri)
+      assert.equal(appsByName?.[0]?.category, newAppMetadataOwnedByMember.category)
+      assert.equal(appsByName?.[0]?.description, updatedMetadata.description)
+      assert.equal(appsByName?.[0]?.oneLiner, updatedMetadata.oneLiner)
+      assert.deepEqual(appsByName?.[0]?.platforms, updatedMetadata.platforms)
+    }
+  )
+
+  // app created by lead
+
+  const leadId = await api.getLeaderId('contentWorkingGroup')
+
+  const newAppNameOwnedByLead = 'update_app_owned_by_lead'
+  const newAppOwnedByLeadMetadata: Partial<AppMetadata> = {
+    category: 'blockchain',
+    oneLiner: 'best blokchain video platform',
+    description: 'long description',
+    platforms: ['web', 'mobile'],
+  }
+
+  await api.createApp(leadId, newAppNameOwnedByLead, newAppOwnedByLeadMetadata, true)
+
+  const appsCreatedByLead = await query.tryQueryWithTimeout(
+    () => query.getAppsByName(newAppNameOwnedByLead),
+    (appsByName) => {
+      assert.equal(appsByName?.[0]?.name, newAppNameOwnedByLead)
+      assert.equal(appsByName?.[0].ownerMember?.id, null)
+      assert.equal(appsByName?.[0].isLeadOwned, true)
+      assert.equal(appsByName?.[0]?.category, newAppOwnedByLeadMetadata.category)
+      assert.equal(appsByName?.[0]?.oneLiner, newAppOwnedByLeadMetadata.oneLiner)
+      assert.equal(appsByName?.[0]?.description, newAppOwnedByLeadMetadata.description)
+      assert.equal(appsByName?.[0]?.termsOfService, null)
+      assert.equal(appsByName?.[0]?.websiteUrl, null)
+      assert.deepEqual(appsByName?.[0]?.platforms, newAppOwnedByLeadMetadata.platforms)
+    }
+  )
+
+  const newAppOwnedByLeadId = appsCreatedByLead?.[0]?.id
+  if (newAppOwnedByLeadId) {
+    await api.updateApp(leadId, newAppOwnedByLeadId, updatedMetadata, true)
+  }
+
+  await query.tryQueryWithTimeout(
+    () => query.getAppsByName(newAppNameOwnedByLead),
+    (appsByName) => {
+      assert.equal(appsByName?.[0]?.id, newAppOwnedByLeadId)
+      assert.equal(appsByName?.[0]?.name, newAppNameOwnedByLead)
+      assert.equal(appsByName?.[0]?.ownerMember?.id, null)
+      assert.equal(appsByName?.[0]?.useUri, updatedMetadata.useUri)
+      assert.equal(appsByName?.[0]?.category, updatedMetadata.category)
```

Comment:
> I don't know why tests are failing. Tried locally, and everything seems fine to me - tests are passing.

Error from the workflow (https://github.com/Joystream/joystream/actions/runs/4133895124/jobs/7144408740): 
> 2023-02-09T12:19:00.8946237Z 2023-02-09T12:19:00.893Z integration-tests:job:update app flow 0 failed:
2023-02-09T12:19:00.9697378Z AssertionError: expected 'blockchain' to equal undefined
2023-02-09T12:19:00.9698158Z     at /home/runner/work/joystream/joystream/tests/network-tests/src/flows/content/updateApp.ts:105:14

The `updatedMetadata` only specifies `description`, `oneLiner` and `platforms`, so other properties should remain intact (just like in case of the member app):

```suggestion
      assert.equal(appsByName?.[0]?.useUri, newAppOwnedByLeadMetadata.useUri)
      assert.equal(appsByName?.[0]?.category, newAppOwnedByLeadMetadata.category)
```